PowerEditor/src/WinControls/AboutDlg/AboutDlg.cpp

```diff
@@ -145,171 +145,8 @@ intptr_t CALLBACK DebugInfoDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM
 	{
 		case WM_INITDIALOG:
 		{
-			NppParameters& nppParam = NppParameters::getInstance();
-
 			NppDarkMode::autoSubclassAndThemeChildControls(_hSelf);
 
-			// Notepad++ version
-			_debugInfoStr = NOTEPAD_PLUS_VERSION;
-			_debugInfoStr += nppParam.archType() == IMAGE_FILE_MACHINE_I386 ? TEXT("   (32-bit)") : (nppParam.archType() == IMAGE_FILE_MACHINE_AMD64 ? TEXT("   (64-bit)") : TEXT("   (ARM 64-bit)"));
-			_debugInfoStr += TEXT("\r\n");
-
-			// Build time
-			_debugInfoStr += TEXT("Build time : ");
-			generic_string buildTime;
-			WcharMbcsConvertor& wmc = WcharMbcsConvertor::getInstance();
-			buildTime += wmc.char2wchar(__DATE__, CP_ACP);
-			buildTime += TEXT(" - ");
-			buildTime += wmc.char2wchar(__TIME__, CP_ACP);
-			_debugInfoStr += buildTime;
-			_debugInfoStr += TEXT("\r\n");
-
-#if defined(__GNUC__)
-			_debugInfoStr += TEXT("Built with : GCC ");
-			_debugInfoStr += wmc.char2wchar(__VERSION__, CP_ACP);
-			_debugInfoStr += TEXT("\r\n");
-#elif !defined(_MSC_VER)
-			_debugInfoStr += TEXT("Built with : (unknown)\r\n");
-#endif
-
-			// Binary path
-			_debugInfoStr += TEXT("Path : ");
-			TCHAR nppFullPath[MAX_PATH];
-			::GetModuleFileName(NULL, nppFullPath, MAX_PATH);
-			_debugInfoStr += nppFullPath;
-			_debugInfoStr += TEXT("\r\n");
-
-			// Command line as specified for program launch
-			_debugInfoStr += TEXT("Command Line : ");
-			_debugInfoStr += nppParam.getCmdLineString();

```

Comment:
Use `$COMMAND_LINE_PLACEHOLDER$` as place holder.

Reply:
As suggested, I am using the `$COMMAND_LINE_PLACEHOLDER$` for the command line parameter. But I have abstracted 
 the `$COMMAND_LINE_PLACEHOLDER$` string into `const _cmdLinePlaceHolder`. This abstraction allows a single definition of this placeholder string value to be used in multiple places: placeholder insertion in WM_INITDIALOG, find placeholder position and replace placeholder in refreshDebugInfo().